src/Microsoft.AspNet.Session/SessionServiceCollectionExtensions.cs

```diff
@@ -7,8 +7,16 @@
 
 namespace Microsoft.Framework.DependencyInjection
 {
+    /// <summary>
+    /// Extension methods for adding session services to the DI container.
+    /// </summary>
     public static class SessionServiceCollectionExtensions
     {
+        /// <summary>
+        /// Adds services the session state middleware.
```

Comment:
Sorry, one more issue - I'm not sure what this doc comment is trying to say (it has a grammatical issue, but I can't understand the intention of the doc).